source/org/zfin/marker/SequenceFeature.java

```diff
@@ -13,33 +12,8 @@ public class SequenceFeature {
     @JsonView({View.API.class, View.ExpressedGeneAPI.class, View.UI.class})
     public String zdbID;
 
-    public String getNameOrder() {

```

Comment:
Did you add a lombok @Data or @Getter and @Setter?

Reply:
The "@Getter" and "@Setter"  were already there, but some of the methods were redundant.